### ISSUE TITLE
Add a method to include dropout during training

### ISSUE THREAD

**Issue by AOS55** [AOS55/Fixedwing-Airsim#29]
Dropout is important for regularization add the PyTorch method somewhere https://jmlr.org/papers/volume15/srivastava14a.old/srivastava14a.pdf
